CTF spawner awareness in tips (#57995)
tgstation/tgstation: strings/tips.txt

As a Drone, you can ping other drones to alert them of areas in the station in need of repair.
As a Ghost, you can see the inside of a container on the ground by clicking on it.
As a Ghost, you can double click on just about anything to follow it. Or just warp around!

> As a Ghost, you can both start and join the capture the flags games by clicking on either the red or blue spawner, which can be found under the "Misc" section of the orbit menu.

Reviewer: ```suggestion
As a Ghost, you can both start and join capture the flag games by clicking on either the red or blue spawner, which can be found under the "Misc" section of the orbit menu.
```

> As a Ghost, you can join the CTF (capture the flag) games via clicking on either the red or blue spawner (which can be always be found under misc section of the ghost orbit menu).

Reviewer: ```suggestion
As a Ghost, you can both start and join the capture the flags games by clicking on either the red or blue spawner, which can be found under the "Misc" section of the orbit menu.
```
